### ISSUE TITLE
only camera test mAP==0?

### ISSUE THREAD

**Issue by mayberpf** [ADLab-AutoDrive/BEVFusion#76]
hello ,thanks for your code.
We used your code for step-by-step training. In the test, we only tested the camera stream and found that the result of the test was mAP=0. 
We trained using the pre-trained weights you provided, but the test map for the camera stream is 0, but the test for the point cloud stream is fruitful.
In the meantime, we downloaded the camera stream weights you provided for testing, and the result was OK.
Are which parameters are incorrect during training? We have not changed the config file you provided.

**Comment by bodhiyang** [comment]
I encountered the same problem that the loss suddenly increased during 12 epoch and the test result is mAP=0. Do u find the reason causing this problem now?

**Comment by ZJUnewer** [comment]
I encountered the same problem that mAP = 0, do you solve it ?

**Comment by Da1symeeting1** [comment]
I encountered the same problem that mAP = 0, do you solve it ?plzzzz
